<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddressSelectionDialogData</class>
 <widget class="QDialog" name="AddressSelectionDialogData">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>395</width>
    <height>406</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Address Selection Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Multiples results found for address :</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="addressLabel">
     <property name="text">
      <string/>
     </property>
     <property name="textInteractionFlags">
      <set>Qt::TextSelectableByMouse</set>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>Pick one of them in the list below and press Ok</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QListWidget" name="resultsList"/>
   </item>
   <item>
    <widget class="QCheckBox" name="rememberChoiceCB">
     <property name="text">
      <string>Remember my choice</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="okButton">
       <property name="text">
        <string>Ok</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
